src/smartpeak/source/ui/FilePicker.cpp

```diff
@@ -64,56 +64,65 @@ namespace SmartPeak
     static char selected_filename[256] = {0};
     // Folder content / Navigation
     ImGui::BeginChild("Content", ImVec2(1024, 400));
-    // Start of Columns - content_columns
-    ImGui::Columns(4, "content_columns");
-    ImGui::Separator();
-    ImGui::Text("Name"); ImGui::NextColumn();
-    ImGui::Text("Size"); ImGui::NextColumn();
-    ImGui::Text("Type"); ImGui::NextColumn();
-    ImGui::Text("Date Modified"); ImGui::NextColumn();
-    ImGui::Separator();
-
-    for (int i = 0; static_cast<size_t>(i) < pathname_content_[0].size(); ++i)
+    
+    const int column_count = 4;
+    const char* column_names[column_count] = { "Name", "Size", "Type", "Date Modified" };
+    static ImGuiTableColumnFlags column_flags[column_count] = { ImGuiTableColumnFlags_DefaultSort | ImGuiTableColumnFlags_NoHide, ImGuiTableColumnFlags_NoHide, ImGuiTableColumnFlags_NoHide, ImGuiTableColumnFlags_NoHide
+    };
+    
+    static ImGuiTableFlags table_flags = ImGuiTableFlags_Resizable | ImGuiTableFlags_Hideable | ImGuiTableFlags_Reorderable |
+        ImGuiTableFlags_NoSavedSettings | ImGuiTableFlags_Sortable | ImGuiTableFlags_MultiSortable;
```

Comment:
Would you mind adding in `ImGuiTableFlags_NoBordersInBody` to the table flags?  The tables version we are using has a not so nice border in my opinion :)

Reply:
Thanks for your suggestions, it looks better and simpler without the borders :)